.github/CONTRIBUTING.rst

```diff
@@ -195,33 +195,23 @@ account. To set this up please go to this
 
 How and which tests are executed is controlled by a environment variable
 ``QISKIT_TESTS``. The options are (where ``uc_available = True`` if the user
 credentials are available, and ``False`` otherwise):
 
-+-------------------+--------------------------------------------------------------------------------------------------------------------+-----------------------+--------------------------------------------------+
-|  Option           | Description                                                                                                        | Default               |  If ``True``, forces                             |
-+===================+====================================================================================================================+=======================+==================================================+
-| ``skip_online``   | Skips tests that require remote requests (also, no mocked information is used). Does not require user credentials. | ``False``             | ``rec = False``                                  |
-+-------------------+--------------------------------------------------------------------------------------------------------------------+-----------------------+--------------------------------------------------+
-| ``mock_online``   | It runs the online tests using mocked information. Does not require user credentials.                              | ``not uc_available``  | ``skip_online = False``                          |
-+-------------------+--------------------------------------------------------------------------------------------------------------------+-----------------------+--------------------------------------------------+
-| ``run_slow``      | It runs tests tagged as *slow*.                                                                                    | ``False``             |                                                  |
-+-------------------+--------------------------------------------------------------------------------------------------------------------+-----------------------+--------------------------------------------------+
-| ``rec``           | It records the remote requests. It requires user credentials.                                                      | ``False``             | ``skip_online = False``                          |
-|                   |                                                                                                                    |                       | ``run_slow = False``                             |
-+-------------------+--------------------------------------------------------------------------------------------------------------------+-----------------------+--------------------------------------------------+
++-----------------+--------------------------------------------------------------------------------------------------------------------+-----------+
```

Comment:
Can the `uc_available` comment above be removed?

Reply:
Yep